### PR TITLE
Send Prebid Version With Every Request

### PR DESCRIPTION
**Pull request by wi101**
## [CM-971](https://liveintent.atlassian.net/browse/CM-971)

[CM-971]: https://liveintent.atlassian.net/browse/CM-971?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: test/spec/modules/liveIntentIdSystem_spec.js [hunk 1]
```diff
@@ -89,6 +89,9 @@ describe('LiveIntentId', function() {
       emailHash: '58131bc547fb87af94cebdaf3102321f'
     }});
     setTimeout(() => {
+      const packageJson = require('../../../package.json')
+      const version = packageJson.version
+      expect(server.requests[0].url).to.contain(`tv=${version}`)
```
Comment (3link) [inline test/spec/modules/liveIntentIdSystem_spec.js:94]: This test can be separated from the others. That way the test scopes will be clearer.

File: test/spec/modules/liveIntentIdSystem_spec.js [hunk 2]
```diff
@@ -83,17 +95,6 @@ describe('LiveIntentId', function() {
...
-    liveIntentIdSubmodule.getId({ params: {
-      ...defaultConfigParams,
-      emailHash: '58131bc547fb87af94cebdaf3102321f'
-    }});
-    setTimeout(() => {
-      expect(server.requests[0].url).to.match(/https:\/\/rp.liadm.com\/j\?.*e=58131bc547fb87af94cebdaf3102321f.+/)
-      done();
-    }, 200);
-  });
-
```
Comment (3link) [inline test/spec/modules/liveIntentIdSystem_spec.js:96]: That test probably should stay 😄

Comment (wi101) [reply test/spec/modules/liveIntentIdSystem_spec.js:96]: 🤦‍♀️  it was long time, and forgot that I updated an existing test case
thanks!

**Comment by wi101** [comment]
getested on staging

**Review by 3link — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -134,7 +134,7 @@
     "express": "^4.15.4",
     "fun-hooks": "^0.9.9",
     "just-clone": "^1.0.2",
-    "live-connect-js": "^6.3.2"
+    "live-connect-js": "^6.3.4"
```
Comment (3link) [inline package.json:137]: Unrelated, but we should talk about it soon: `^` means, when you install the dependencies, the build system will automatically check for a newer version 6.x.y. That mean when we release e.g. a new patch of LiveConnect, but do not want it to be rolled out yet, but want to test it with a limited number of users, Prebid might just run their build and the newer version will be pulled in - something that we did not intend.

**Comment by wi101** [comment]
opened a [PR](https://github.com/prebid/Prebid.js/pull/10920) in Prebid repo

**Pull request closed without merging**